### PR TITLE
Functional V0.3 (Images will be added after PR validation)

### PR DESCRIPTION
**Pull request by PortetWilfried**

### CODE REVIEW

File: documents/functional_specifications/Functional Specification.md [hunk 1]
```diff
@@ -23,34 +21,38 @@
...
 
 ## 2. Overview
 
-The aim of this document is to provide a comprehensive overview of the "Adopt a Candidate" application. This application serves as a bridge between job seekers and companies, with a unique feature of anonymity to prevent any form of discrimination. The app focuses on matching soft skills rather than qualifications. The project's scope involves the development of key features such as user account creation, a matching algorithm, and job posting management. This document will serve as a roadmap, detailing each facet of the application for clarity and understanding.
+This document aims to provide a comprehensive overview of the "Adopt a Candidate" application. This application serves as a bridge between jobseekers and companies, with a unique feature of anonymity to prevent any form of discrimination. The app focuses on matching soft skills rather than qualifications. The project's scope involves the development of key features such as user account creation, a matching algorithm, and job posting management. This document will serve as a roadmap, detailing each facet of the application for clarity and understanding.
```
Comment (EnzoGuillouche) [inline documents/functional_specifications/Functional Specification.md]: Job-seekers should contain a dash.

File: documents/functional_specifications/Functional Specification.md [hunk 2]
```diff
@@ -71,35 +73,50 @@ Developed using Flutter 3, and commissioned by "We Are Evolution," this applicat
...
+- Jobseekers will be able to search for soft skills through a search bar.
+    - In case the soft skills they want to add don't exist in the database, they can create it.
+- Allow companies to specify the soft skills they are looking for via checkboxes or search bar.
 
 ### 2.3 User Roles
 
 **Intended users of the software:**
 
-- **Job Seekers:** Citizens seeking employment, whether skilled or low-skilled, by highlighting their soft skills rather than formal qualifications. The goal is to help job seekers find positions that match their soft skills.
+- **Job Seekers:** Citizens seeking employment, whether skilled or low-skilled, by highlighting their soft skills rather than formal qualifications. The goal is to help jobseekers find positions that match their soft skills.
```
Comment (EnzoGuillouche) [inline documents/functional_specifications/Functional Specification.md]: Same for job-seekers.

File: documents/functional_specifications/Functional Specification.md [hunk 3]
```diff
@@ -71,35 +73,50 @@ Developed using Flutter 3, and commissioned by "We Are Evolution," this applicat
...
 1. Download the application.
 2. Select the preferred language for the application's text.
 3. Create an account by providing personal information.
-4. Select relevant soft skills via checkboxes.
+4. Select relevant soft skills via the search bar.
 5. Specify the geographical area for job searching.
 6. Access the homepage to view companies currently hiring for the selected soft skills.
-7. On the homepage the job seeker must like the company to initiate the match.
-8. If a company expresses interest in the jobseekers profile, the connection process begins.
+7. On the homepage the jobseeker must like the company to initiate the match.
```
Comment (EnzoGuillouche) [inline documents/functional_specifications/Functional Specification.md]: Same for job-seeker.

File: documents/functional_specifications/Functional Specification.md [hunk 4]
```diff
@@ -71,35 +73,50 @@ Developed using Flutter 3, and commissioned by "We Are Evolution," this applicat
...
+8. Then the jobseeker needs to wait for a company to start a conversation.
```
Comment (EnzoGuillouche) [inline documents/functional_specifications/Functional Specification.md]: Again.

File: documents/functional_specifications/Functional Specification.md [hunk 5]
```diff
@@ -71,35 +73,50 @@ Developed using Flutter 3, and commissioned by "We Are Evolution," this applicat
...
+
+**Example: Jobseeker's experience**
```
Comment (EnzoGuillouche) [inline documents/functional_specifications/Functional Specification.md]: This is the last comment I write about job-seekers, look throughout the file for this mistake.

**Review by EnzoGuillouche — CHANGES_REQUESTED** [review]
Some changes are requested.

File: documents/functional_specifications/Functional Specification.md (hunk 1 above)
Comment (PortetWilfried) [reply documents/functional_specifications/Functional Specification.md]: I checked the spelling for jobseeker and it's jobseeker not job-seeker.

**Review by EnzoGuillouche — APPROVED** [review]
Merging approved.

**Review by Mattstar64 — APPROVED** [review]
Everything seems in order and all documentation criteria are checked. You can merge !

**Review by NanaChocolat — APPROVED** [review]

**Pull request merged**